cli/src/opts/cast.rs

```diff
@@ -298,6 +298,19 @@ pub enum Subcommands {
         #[structopt(subcommand)]
         command: WalletSubcommands,
     },
+    #[structopt(name = "interface", about = "generate contract's interface from ABI")]
+    Interface {
+        #[structopt(short = "s", help = "The contract's address")]
+        contract_address: Option<String>,
```

Comment:
The CLI args need to be adjusted to match the "local or etherscan" logic I think